plugins/procedural/main.cpp

```diff
@@ -46,10 +46,17 @@ inline ProceduralReader *CreateProceduralReader(AtUniverse *universe)
 {
 #ifdef ENABLE_HYDRA_IN_USD_PROCEDURAL
     if (ArchHasEnv("PROCEDURAL_USE_HYDRA")) {
-        return new HydraArnoldReader(universe);
-    } else {
-        return new UsdArnoldReader();
+        std::string useHydra = ArchGetEnv("PROCEDURAL_USE_HYDRA");
+        std::string::size_type i = useHydra.find(" ");
```

Comment:
I was thinking more of testing just the first character like

if (!useHydra.empty && useHydra[0] == '0') {
  // use usd reader
}